docs/get-started/_index.md

```diff
@@ -16,32 +16,27 @@ images: ["/general/understand.png"]
 carouselscript: true
 ---
 
-Machines are everywhere, from small machines like IoT sensors, to home automation systems, robotic systems, cars and boats, and even more complex enterprise systems.
-All these machines start by combining hardware and software.
-
 Viam is a software platform that makes it easy to combine and integrate hardware and software to build machines, connect them with the cloud, and make them smarter with machine learning.
 
 <img src="../about-viam.png" width="800px" alt="Viam architecture on device where viam-server uses the machine configuration to manage drivers for different hardware and in the cloud where the platform offer data management, ML, and extended functionality.">
 
-## `viam-server`
+## Support
 
-At the core of Viam is the open-source `viam-server` executable which runs on a computer and provides drivers for hardware and manages software and data for a machine.
-If you are working with microcontrollers, [`viam-micro-server`](/installation/) is a lightweight version of `viam-server` which can run on resource-limited embedded systems that cannot run the fully-featured `viam-server`.
+Viam runs on **microcontrollers** and **64-bit Linux OS, macOS, or Windows Subsystem for Linux**.
 
-To use Viam with a machine, you create a configuration specifying which hardware and software the machine consists of.
-Viam has many built-in {{< glossary_tooltip term_id="component" text="components" >}} and {{< glossary_tooltip term_id="service" text="services" >}} that run within `viam-server`.
-The components and services are configurable building blocks you can put together to make your machine.
-`viam-server` then manages and runs the drivers for the configured hardware components and software services.
+{{< board-carousel >}}
+<br>
 
-For example, if you are building a pet-feeding machine you might install `viam-micro-server` on a microcontroller and create a machine configuration with a camera and a servo for the hardware you are using.
+## Install Viam on your machine
```

Comment:
This heading doesn't quite fit the contents--"install" isn't the same as use APIs etc. Maybe just this??
```suggestion
## Viam on your machine
```